react/Infos/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+import styles from './styles.styl'
+import Icon from '../Icon'
+import Text from '../Text'
+const Infos = ({ icon, text, className }) => {
+  return (
+    <div className={cx(styles['infos'], 'u-p-1', className)}>
+      <div className={styles['infos--container']}>
+        {icon && <Icon icon={icon} className={styles['infos--icon']} />}
+        <Text
+          className={cx(styles['infos--text'], {
+            ['u-pl-half']: icon !== null
+          })}
+        >
+          {text}
+        </Text>
+      </div>
+    </div>
+  )
+}
+
+Infos.defaultProps = {
+  icon: null
+}
+Infos.prototype = {
+  icon: PropTypes.any,
```

Comment:
Is there a reason for not using the same propTypes as in `Icon` here ? Cf. https://github.com/cozy/cozy-ui/blob/master/react/Icon/index.jsx#L69 (minus `isRequired` obviously ;))

Reply:
You're right. I added this any at the beginning and didn't change it. 

`isRequired` is not required since we can use this component without icon see https://zpl.io/V0JGPwx